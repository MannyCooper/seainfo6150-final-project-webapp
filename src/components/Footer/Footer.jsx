import React from "react";
import { Link } from "react-router-dom";
import styles from "./Footer.module.css";
import portrait from "../../images/portrait.png";

const Footer = () => {
    return (
        <footer className={styles.footer}>
            <Link to="/">
                <img className={styles.logo} src={portrait} alt="portrait"/>
            </Link>
            <nav className={styles.nav}>
                <span className={styles.copyright}>Copyright Â© 2021</span>
                <Link className={styles.link} to="/">Zifan An's Blog</Link>
            </nav>
        </footer>
        )
}

export default Footer;