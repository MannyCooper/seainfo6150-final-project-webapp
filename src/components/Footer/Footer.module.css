.footer {
    max-width: 1100px;
    margin: 0 auto;
}

.nav {
    display: flex;
    margin-top: 3px;
    border-top: 1px solid #d2d2d7;
    color: #86868b;
    margin-top: 10px;
    padding-bottom: 40px;
}

.copyright,
.link {
    font-size: 14px;
}

.link {
    margin-left: 20px;
}

.logo {
    margin-top: 15px;
    height: 32px;
    width: 32px;
    border-radius: 60px
}

@media only screen and (max-width: 1380px){
    .footer{
        max-width: 980px;
    }
}

@media only screen and (max-width: 1200px){
    .footer{
        width: 692px;
    }
}

@media only screen and (max-width: 768px){
    .footer{
        max-width: 366px;
        width: 87.5%;
    }
}
