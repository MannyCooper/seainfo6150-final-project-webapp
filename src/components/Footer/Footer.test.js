 /* global render */
 import React from "react";
 import Footer from "./Footer.jsx";
  
 it("renders the Footer component correctly", () => {
   const { container } = render(<Footer />);
   expect(container).toMatchSnapshot();
 });
 