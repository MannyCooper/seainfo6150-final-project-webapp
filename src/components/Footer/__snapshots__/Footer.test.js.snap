// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Footer component correctly 1`] = `
<div>
  <footer
    class="footer"
  >
    <a
      href="/"
    >
      <img
        alt="portrait"
        class="logo"
        src="portrait.png"
      />
    </a>
    <nav
      class="nav"
    >
      <span
        class="copyright"
      >
        Copyright Â© 2021
      </span>
      <a
        class="link"
        href="/"
      >
        Zifan An's Blog
      </a>
    </nav>
  </footer>
</div>
`;
